src/modules/operations.js

```diff
@@ -0,0 +1,66 @@
+/*eslint-disable*/
+export  class Operations {
+  static createTask() {
+    const inputData = document.getElementById('inputText').value;
+    const saveData = document.getElementById('formText');
+    const storage = this.getAllTasks().length;
+    const task = [{ description: inputData, completed: false, index: storage }];
+    this.saveTask(task);
+    this.showTask();
+    saveData.reset();
+  }
```

Comment:
- [ ] When a task is added its value should be equal to its position on the array, starting from count 1. That means the first task should have an `index` of 1, and the 13th task should have an `index` of 13. Your index value starts at 0, kindly fix that.